pypeit/spectrographs/vlt_fors.py

```diff
@@ -259,18 +278,18 @@ def get_detector_par(self, det, hdu=None):
             specflip        = False,
             spatflip        = False,
             platescale      = 0.126,  # average between order 11 & 30, see manual
-            darkcurr        = 0.0,
+            darkcurr        = 1.4,
             saturation      = 2.0e5,  # I think saturation may never be a problem here since there are many DITs
             nonlinear       = 0.80,
             mincounts       = -1e10,
             numamplifiers   = 1,
             gain            = np.atleast_1d(0.70),
-            ronoise         = np.atleast_1d(3.0),  # High gain
-            datasec         = np.atleast_1d('[20:,0:2048]'),
-            oscansec        = np.atleast_1d('[4:20,4:2044]'),
-            #suffix          = '_Belenos'
+            ronoise         = np.atleast_1d(3.15),  # High gain
+            datasec=np.atleast_1d('[11:2059,:]'),
```

Comment:
This is a pretty significant change..
Are you sure it is right?

Reply:
I'm not completely sure, but the tests will inform us. I believe I added this because the old numbers were not correct.